package-lock.json

```diff
@@ -3608,9 +3558,9 @@
       }
     },
     "node_modules/eslint-plugin-react-hooks": {
-      "version": "4.6.2",
-      "resolved": "https://registry.npmjs.org/eslint-plugin-react-hooks/-/eslint-plugin-react-hooks-4.6.2.tgz",
-      "integrity": "sha512-QzliNJq4GinDBcD8gPB5v0wh6g8q3SUi6EFF0x8N/BL9PoVs0atuGc47ozMRyOWAKdwaZ5OnbOEa3WR+dSGKuQ==",
+      "version": "5.0.0-canary-7118f5dd7-20230705",
```

Comment:
Is only the canary available?

Reply:
Again, it's due to `eslint-config-next` asking for this version directly.
https://github.com/vercel/next.js/blob/27554305bc2687cbbae651c79e8d9e9d8ae7bac1/packages/eslint-config-next/package.json#L22
I suppose I could try rolling it back to `4.x` - but with `eslint-config-next` being updated to `15.x` in #341 anyway, this will be resolved, because that series asks for `5.0.0` proper.

Reply:
Nah, this is fine, I was just curious. If it's being fixed properly soon anyway there's no real issue.